layersvt/api_dump.h

```diff
@@ -336,6 +336,9 @@ class ApiDumpSettings {
         std::string filename_string = "";
         // If the layer settings file has a flag indicating to output to a file,
         // do so, to the appropriate filename.
+
+        // getLayerOption is defined inside VVL via vk_layer_config.cpp / vk_layer_config.h
+        // How should this be updated?
         const char *file_option = getLayerOption("lunarg_api_dump.file");
```

Comment:
@christophe-lunarg on how to procede.

I thought about copying vk_layer_config.cpp from VVL but that solution didn't seem correct while I was implementing it.